ui-react/packages/atlasmap-ui/src/mapper/MapperContextToolbar.tsx

```diff
@@ -15,13 +15,16 @@ import {
 } from '@patternfly/react-core';
 
 import { FilePicker } from 'react-file-picker';
-import { processImportedFile } from './MapperUtilsToolbar';
+// import { processImportedFile } from './MapperUtilsToolbar';
 
-export interface IMapperContextToolbarProps {}
+
+export interface IMapperContextToolbarProps {
+  importAtlasFile: (selectedFile: File) => void;
+}
 
 export const MapperContextToolbar: FunctionComponent<
   IMapperContextToolbarProps
-> = () => {
+> = (importAtlasFile) => {
```

Comment:
This should be `({ importAtlasFile }) => {`, which uses [ES6 object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) to save some keystroke.

The extended version would be like this:

```tsx
export const MapperContextToolbar: FunctionComponent<IMapperContextToolbarProps> = (props) => {
  const importAtlasFile = props.importAtlasFile;
```

The former is the preferred way to do things in react-land.